apps/trading/client-pages/portfolio/deposits-container.tsx

```diff
@@ -20,14 +19,10 @@ export const DepositsContainer = () => {
           }}
         />
       </div>
-      <DepositDialog
-        depositDialog={depositDialog}
-        setDepositDialog={setDepositDialog}
-      />
       <div className="w-full dark:bg-black bg-white absolute bottom-0 h-auto flex justify-end px-[11px] py-2">
         <Button
           size="sm"
-          onClick={() => setDepositDialog(true)}
+          onClick={() => openDepositDialog()}
```

Comment:
suggestion (non blocking): no need for new function
```suggestion
          onClick={openDepositDialog}
```